doc/build-with-vs.md

```diff
@@ -1,77 +1,67 @@
 Build with Visual Studio
```

Comment:
It would be great to be able to reproduce this instructions in a Windows docker worked on in PR #114.